spec/factories/kudos.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :kudo do
+    title { 'Title' }
+    content { 'Content' }
+    association :giver_of_kudo, factory: :employee
+    association :receiver_of_kudo, factory: :employee
```

Comment:
[FYI] I think it is possible to write simply:
giver_of_kudo
receiver_of_kudo
but it is simply matter of preference - if you like it in explicit way it's fine ;)